migrationtools/databases.py

```diff
@@ -40,9 +89,75 @@ def _setup_tables(self, meta):
 
 class SeedIdentity(Database):
 
+    # The ID of the account to create records as
+    created_by_id = 1
+    # The ID of the account to update records as
+    updated_by_id = 1
```

Comment:
This could already be an existing user in the system, should we do a `get_or_create()` type thing here for a "migrator account" ?